### PR TITLE
Remove DOCUMENTATION_FILE toggle

### PR DESCRIPTION
**Pull request by benrudolph**

### CODE REVIEW

**Comment by benrudolph** [comment]
Hold off on merge, need to make sure arya is OK with not adding a second license dropdown for the documentation file

File: corehq/apps/domain/forms.py [hunk 1]
```diff
@@ -186,15 +186,13 @@ def __init__(self, *args, **kw):
...
             'image',
+            'documentation_file',
             'video',
             'share_multimedia',
             'share_reminders',
             'license',
             'cda_confirmed',]
 
-        if self.dom and toggles.DOCUMENTATION_FILE.enabled(user.username):
-            self.fields.keyOrder.insert(5, 'documentation_file')
```
Comment (dannyroberts) [inline corehq/apps/domain/forms.py]: Ha! Talk about fragile :)

**Comment by dannyroberts** [comment]
Looks good to review! (Was super easy once I added ?w=1 to the url to ignore whitespace in the diff)

File: corehq/apps/domain/forms.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/domain/forms.py]: lol yup, figured though it was the easiest thing to remove when we removed the toggle

**Pull request closed without merging**